go/service/chat_local.go

```diff
@@ -63,6 +63,22 @@ func (h *chatLocalHandler) NewConversationLocal(ctx context.Context, trip chat1.
 	return id, err
 }
 
+func (h *chatLocalHandler) CompleteAndCanonicalizeTlfName(ctx context.Context, tlfName string) (res keybase1.CanonicalTlfName, err error) {
+	// Append username in case it's not present. We don't need to check if it
+	// exists already since CryptKeys calls below transforms the TLF name into a
+	// canonical one.
+	tlfName = tlfName + "," + string(h.G().Env.GetUsername())
```

Comment:
It seems that even if the user is not logged in, it still returns the username here, but fails below when calling KBFS service and error message says logging in is required. But I'll add a check here in case user's device is not provisioned. Thanks Max!